docs/ref/contrib/postgres/fields.txt

```diff
@@ -7,6 +7,33 @@ module.
 
 .. currentmodule:: django.contrib.postgres.fields
 
+``SerialField``
+===============
+
+.. versionadded:: 1.11
+
+.. class:: SerialField(unique=True)
+
+    A field similar to the :class:`AutoField` except it does not need to be
+    a primary key. It uses a separate postgres sequence to automatically
+    increment the integer value.
+
+    .. attribute:: base_field
+
+        This is an optional argument.
+
+        The field is unique by default, however a serial does not need to be unique.
+
+``BigSerialField``
+===============
+
+.. versionadded:: 1.11
+
+.. class:: BigSerialField(unique=True)
+
+    A 64-bit integer, much like an :class:`SerialField` except that it is
+    guaranteed to fit numbers from ``1`` to ``9223372036854775807``.
```

Comment:
@charettes @francoisfreitag as much as this might be `2**61-1` it is actually not documented in the Postgres documentation. Therefore I will just keep it as is.